### PR TITLE
feat: Upgrade to latest ffresty with mTLS

### PR DESCRIPTION
**Pull request by EnriqueL8**
Depends on:
- https://github.com/hyperledger/firefly-common/pull/65
- https://github.com/hyperledger/firefly/pull/1289

Once those are in I'll update this one

### CODE REVIEW

File: internal/blockchain/ethereum/address_resolver.go
Comment (peterbroadhurst) [inline internal/blockchain/ethereum/address_resolver.go:62]: Looks like an extra test is needed to retain coverage and cover this branch.

Something like: https://github.com/hyperledger/firefly-signer/blob/bece7de7dc20d55bc99517a8255c2ec93f596581/internal/rpcserver/server_test.go#L63-L75

**Review by peterbroadhurst — CHANGES_REQUESTED** [review]
Changes look great @EnriqueL8

If you wouldn't mind including the corresponding tests for coverage as well before we merge.

Not sure why the code coverage bot didn't run on your branch to help with that 🤔 

Toggling code coverage in your vscode for each of the affected packages should help though.

**Comment by EnriqueL8** [comment]
@peterbroadhurst let's hold off on this one until I get the wsclient changes in

File: internal/blockchain/ethereum/config.go
Comment (peterbroadhurst) [inline internal/blockchain/ethereum/config.go:77]: Why is this now needed @EnriqueL8?

Comment (EnriqueL8) [reply internal/blockchain/ethereum/config.go:77]: So we hide that underneath the init config from the wsclient since `wsClient.InitConfig` calls `ffresty.InitConfig`. So if we ever want to separate the configs for the ws and ffresty we would need to call it here, but since they are the same today I can remove this line

**Review by peterbroadhurst — APPROVED** [review]
Thanks @EnriqueL8

**Pull request merged**
